.eslintignore

```diff
@@ -1,3 +1,4 @@
 /common/component/component.js
 /selfie_segmentation/tflite-support/
-/selfie_segmentation/lib/
\ No newline at end of file
+/selfie_segmentation/lib/
+/nnotepad/js/index.js
```

Comment:
If you add `/* global Monaco */` to the start of `nnotepad/js/index.js` then eslint passes and you can revert this change.

Reply:
Thank you @inexorabletash for the great feedback.

One line code of `/* global monaco */` is much better than several `// eslint-disable-next-line no-undef`, PTAL.

Reply:
Thank you @inexorabletash for the great feedback.

One line code of `/* global monaco */` is much better than several `// eslint-disable-next-line no-undef`, PTAL.